cli/CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
-## 13.6.1
+## 13.7.0
 
 _Released 12/5/2023 (PENDING)_
 
+**Features:**
```

Comment:
This feels like a misc task that users don't care about. I wouldn't flag this as a feature since it's internal to us.

Reply:
updated in https://github.com/cypress-io/cypress/pull/28418/commits/05ac027d55839d670f998984e08d1f8d9e5aa6ad